features/facebookImport/src/views/overview/overview.jsx

```diff
@@ -19,43 +24,65 @@ const Overview = () => {
-                    <p>
-                        {i18n.t("overview:imported.time")} {file.time}
-                    </p>
-                    <p>
-                        {i18n.t("overview:size")} {file.size} bytes
-                    </p>
-                    <div className="btn-holder">
-                        <RouteButton className="btn primary" route="/explore">
-                            {i18n.t("overview:explore")}
-                        </RouteButton>
-                        <RouteButton
-                            className="btn secondary"
-                            route="/import"
-                            stateChange={{
-                                importStatus: importSteps.import,
-                            }}
-                            onClick={() => handleRemoveFile(file.id)}
-                        >
-                            {i18n.t("overview:new.import")}
-                        </RouteButton>
-                    </div>
-                </div>
-            ))}
-        </>
-    );
-
     return (
         <div className="overview">
             {facebookHeader}
             {Object.values(files).length ? (
-                fileList
+                <>
+                    <div className="details">
+                        <p>Name: {files[0].name}</p>
+                        <p>
+                            {i18n.t("overview:imported.time")} {files[0].time}
+                        </p>
+                        <p>
+                            {i18n.t("overview:size")} {files[0].size} bytes
+                        </p>
+                        <div className="separator"></div>
+                    </div>
+
+                    <div className="imported-files">
+                        <h4>{i18n.t("overview:imported.files")}</h4>
+                        {facebookAccount &&
+                        facebookAccount.importedFileNames.length ? (
+                            <div className="file-list">
+                                {facebookAccount.importedFileNames.map(
+                                    (entry, index) => (
+                                        <div
+                                            className="file-button"
+                                            key={index}
+                                        >
+                                            {entry}
+                                        </div>
+                                    )
+                                )}
+                            </div>
+                        ) : (
+                            ""
+                        )}
+                    </div>
+
+                    <div className="footer">
+                        <div className="overlay"></div>
+                        <div className="btn-holder">
+                            <RouteButton
+                                className="btn primary"
+                                route="/explore"
```

Comment:
Isn't there a central place where all these routes are named? As usual, I *really* prefer to avoid literals and use constants.

Reply:
for now I guess all routes are expresses as strings. We could do a PR where very Component that supports routing  is refactored to also export the path alongside the component

Reply:
We could define the routes as top-level constants as it is done with importstatus to avoid strings and carry them through the context

Reply:
@RichardSto  What about defining routes in the same file as the component? For example `import.jsx` could define `export const ImportRoute = "/import";` and then all places that need to use that import `ImportRoute`. That way we also get static dependencies between components.

Reply:
@chisandrei Yeah absolutely prefer that!